README.md

```diff
@@ -63,139 +68,219 @@ Use `#pragma mark -` to categorize methods in functional groupings and protocol/
 ```
 
 ## Spacing
+
+### Indentation
```

Comment:
Markdown specifies that there should be a blank line after a heading. Therefore:

```markdown
### Indentation

* **Indent using 4 spaces.…**
```